openstf/model/ml_model/abstract_ml_model.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from abc import ABC, abstractmethod
+import pandas as pd
+import structlog
+from typing import Union
+import numpy as np
+from sklearn.base import RegressorMixin, BaseEstimator
+
+
+class AbstractMLModel(BaseEstimator, RegressorMixin, ABC):
```

Comment:
Doe nog één keer de check wat het verschil nou precies is tussen 'onze'  requirements en wat al door RegressorMixin wordt geeist